[BUG]: The lesson download script is failing in CI

### Describe the bug

The lesson download script has been failing for several months in CI:

https://github.com/oppia/oppia-android/actions/workflows/lesson_checks.yml

In fact it perhaps never succeeded, but the failures have presumably changed.

### Steps To Reproduce

Try to run the lesson compatibility workflow or wait for its weekly cron job.

### Expected Behavior

It should pass and provide lesson context output.

### Screenshots/Videos

_No response_

### What device/emulator are you using?

_No response_

### Which Android version is your device/emulator running?

_No response_

### Which version of the Oppia Android app are you using?

_No response_

### Additional Context

The CI output isn't very helpful:

```
Prepare all required actions
Run ./.github/actions/set-up-android-bazel-build-environment
Run which java
/usr/bin/java
openjdk version "17.0.17" 2025-10-21
OpenJDK Runtime Environment Temurin-17.0.17+10 (build 17.0.17+10)
OpenJDK 64-Bit Server VM Temurin-17.0.17+10 (build 17.0.17+10, mixed mode, sharing)
Java home: /usr/lib/jvm/temurin-17-jdk-amd64
openjdk version "17.0.17" 2025-10-21
OpenJDK Runtime Environment Temurin-17.0.17+10 (build 17.0.17+10)
OpenJDK 64-Bit Server VM Temurin-17.0.17+10 (build 17.0.17+10, mixed mode, sharing)
Error: Process completed with exit code 1.
```

I suspect this is caused by an old environment setup that we've since changed in develop, but the lesson download script's environment is very old.

--- a/.github/workflows/lesson_checks.yml
+++ b/.github/workflows/lesson_checks.yml
@@ -55,4 +55,5 @@ jobs:
             cache_mode=none \
             $(pwd)/release_assets/cache \
             $(pwd)/release_assets/pinned_versions.textproto \
+            false \
             -Werror

